### ISSUE TITLE
Fine tune from pretrained weight

### ISSUE THREAD

**Issue by JISock** [endernewton/tf-faster-rcnn#262]
I am attempting to add different network to your faster rcnn network. The problem is that the network I am attempting to add uses adam optimizer but it does not train well with the tf.train.MomentumOptimizer which is used in this network.

I have attempted to download the latest model weights for res101 and train further with tf.train.AdamOptimizer with learning rate same as the default learning rate for momentum optimizer which is 0.0001. The result shows that the AP decreases from 74(momentum optimizert) to 63(Adam optimizer). Is it expected or just a matter of learning rate tuning?
